tests/testthat/test-project.R

```diff
@@ -81,3 +81,97 @@ test_that("armadillo.delete_project deletes project", {
 
   stub_registry_clear()
 })
+
+test_that("armadillo.get_projects_info gets all projects and their users", {
+  test_data <- list(
+    list("name" = "lifecycle", "users" = list()),
+    list("name" = "other-project", "users" = list("john", "tommy"))
+  )
+  stub_request("get", uri = "https://test.nl/access/projects") %>%
+    to_return(
+      status = 200,
+      body = "[
+        {
+          \"name\": \"lifecycle\",
+          \"users\": []
+        },
+        {
+          \"name\": \"other-project\",
+          \"users\": [\"john\", \"tommy\"]
+        }
+      ]",
+      headers = list("Content-Type" = "application/json")
+    )
+  res <- armadillo.get_projects_info()
+  expect_equal(res, test_data)
+
+  stub_registry_clear()
+})
+
+test_that("armadillo.get_project_users with a project that has users", {
+  stub_request("get", uri = "https://test.nl/access/projects") %>%
+    to_return(
+      status = 200,
+      body = "[
+        {
+          \"name\": \"lifecycle\",
+          \"users\": []
+        },
+        {
+          \"name\": \"other-project\",
+          \"users\": [\"john\", \"tommy\"]
+        }
+      ]",
```

Comment:
Is this the same data ... DRY ? Can we make it a common variable?

Else : please remove the escaping.